docs/user-guide/federation/federated-services.md

```diff
@@ -0,0 +1,378 @@
+---
+---
+# Kubernetes Federated Services and Cross-Cluster Service Discovery
+
+This guide explains how to use Kubernetes Federated Services to deploy
+a common Service across multiple Kubernetes clusters. This makes it
+easy to achieve cross-cluster service discovery and availibility zone
+fault tolerance for your Kubernetes applications.
+
+
+* TOC
+{:toc}
+
+## Prerequisites
+
+This guide assumes that you have a running Kubernetes Cluster
+Federation installation. If not, then head over to the
+[federation admin guide](/docs/admin/federation/) to learn how to
+bring up a cluster federation (or have your cluster administrator do
+this for you). Other tutorials, for example
+[this one](https://github.com/kelseyhightower/kubernetes-cluster-federation)
+by Kelsey Hightower, are also available to help you.
+
+You are also expected to have a basic
+[working knowledge of Kubernetes](/docs/getting-started-guides/) in
+general, and [Services](/docs/user-guide/services/) in particular.
+
+## Overview
+
+Federated Services are created in much that same way as traditional
+[Kubernetes Services](/docs/user-guide/services/) by making an API
+call which specifies the desired properties of your service. In the
+case of Federated Services, this API call is directed to the
+Federation API endpoint, rather than a Kubernetes cluster API
+endpoint. The API for Federated Services is 100% compatible with the
+API for traditional Kubernetes Services.
+
+Once created, the Federated Service automatically:
+
+1. creates matching Kubernetes Services in every cluster underlying your Cluster Federation,
+2. monitors the health of those service "shards" (and the clusters in which they reside), and
+3. manages a set of DNS records in a public DNS provder (like Google Cloud DNS, or AWS Route 53), thus ensuring that clients
+of your federated service can seamlessly locate an appropriate healthy service endpoint at all times, even in the event of cluster,
+availability zone or regional outages.
+
+Clients inside your federated Kubernetes clusters (i.e. Pods) will
+automatically find the local shard of the Federated Service in their
+cluster if it exists and is healthy, or the closest healthy shard in a
+different cluster if it does not.
+
+## Hybrid Cloud Capabilities
```

Comment:
Sentence casing for headings, not title casing (which is only for page titles). Thus:

`## Hybrid cloud capabilities`

This goes for the entire PR, only gonna mention it once here.